packages/ra-ui-materialui/src/layout/LoadingIndicator.js

```diff
@@ -11,7 +11,7 @@ import RefreshIconButton from '../button/RefreshIconButton';
 const useStyles = makeStyles(
     {
         loader: {
-            margin: 14,
+            margin: 16,
```

Comment:
Why don't use `theme.spacing(2)` ? Could use the `useTheme` hook in the component as well